### ISSUE TITLE
angular-ui-router#0.3.1

### ISSUE THREAD

**Issue by victordidenko** [rpocklin/ui-router-tabs#81]
Hi there!

What about angular-ui-router#0.3.1?
In your bower.json there is dependency to angular-ui-router#^0.2.14, that is not resolved to version 0.3.1 -- does that mean that ui-router-tabs will not work with v0.3.1?

**Comment by rpocklin** [comment]
No, it means it hasn't been tested / guaranteed to work with it.  Since 3 is version up from 2 it may have breaking change (will have to check).  No reason you can't use it - please give feedback.

**Comment by KadKla** [comment]
Hi there!

I really like the package, it helps me a lot! However, I think that it does not yet work together with the versions angular-ui-router 1.0.x, I checked all versions starting from 1.0.x alhpa.1. The problem that I recognized is that the tabs doesn't change anymore when you skip back in the browser...

I would be very greatful if it is possible to fix this, since I need to use the angular-ui-router 1.0.x version in my project.

Thanks in advance for your help!

Best wishes!

**Comment by rpocklin** [comment]
Cool, it's definitely not going to work without some modifications but ill take a look at it.

**Comment by rpocklin** [comment]
I just manually tested and it works fine with `0.3.2` but `1.0.0` hasn't been officially released yet.
